tests/gradients/core/test_hadamard_gradient.py

```diff
@@ -881,7 +881,7 @@ def test_output_shape_matches_qnode_expval_array(self, cost, exp_shape, exp_type
     @pytest.mark.parametrize("cost, exp_shape, exp_type", costs_and_expected_probs)
     def test_output_shape_matches_qnode_probs(self, cost, exp_shape, exp_type, mode):
         """Test that the transform output shape matches that of the QNode."""
-        dev = qml.device("default.qubit", wires=5)
+        dev = qml.device("default.qubit", wires=6)
```

Comment:
@albi3ro adjustment made here:
1. first expected shape of cost10 changed 32->8
2. increased the wire number of testing dev 5->6 for being compatible with cost11